articles/language/type-model.md

```diff
@@ -472,3 +472,70 @@ a valid invocation of `Controlled Rz` (note the parentheses around `0.1, target`
 
 As another example, `CNOT(control, target)` can be implemented as `Controlled X([control], target)`. 
 If a target should be controlled by 2 control qubits (CCNOT), we can use `Controlled X([control1, control2], target)` statement.
+
+### Examples
+
+This example of a Q# operation comes from the [Measurement](https://github.com/Microsoft/Quantum/tree/master/Samples/Measurement) sample. Within operations, we can allocate qubits and use quantum operations on those qubits such as `H` and `X`:
+
+```qsharp
+/// # Summary
+/// Prepares a state and measures it in the Pauli-Z basis.
+operation MeasureOneQubit () : Result {
+        result = Zero;
```

Comment:
I think we still need the `mutable` keyword here to make the declaration that's used in 489.
```suggestion
        mutable result = Zero;
```